ome_zarr/cli.py

```diff
@@ -132,6 +134,11 @@ def main(args: List[str] = None) -> None:
         "--in-place", action="store_true", help="if true, don't write the base array"
     )
     parser_scale.add_argument("--downscale", type=int, default=2)
+    parser_scale.add_argument(
+        "--downsample_z",
```

Comment:
Would you imagine having `download_t` in the future as well? Other options I can imagine are:
 * `--downsample-nd='{"z": 1.5, "x": "2.0"}'
 *  `--downsample=2` (applies to x&y) but `--downsample=z:0.15` would apply to just the one dimension and could be used N times.

Reply:
I don't know that downsampling in T would make sense for *all* types of movies, but it might be useful for some. So we should probably support it if it's not too painful.
And it's certainly possible you'd want to downsample by a different factor in different dimensions.
An alternative to the suggestions above could be more like the spec for shape or chunks. Not sure how much harder this would be to implement:
 - `--downsample=1,1,1.5,2,2`